services/user-svc/internal/organization/organization.go

```diff
@@ -684,3 +743,22 @@ func ChangeMembershipAdminStatus(ctx context.Context, db *gorm.DB, userID uuid.U
 
 	return nil
 }
+
+func IsInOrganization(db *gorm.DB, organizationID uuid.UUID, userID uuid.UUID) (bool, error) {
```

Comment:
```suggestion
func IsInOrganization(db *gorm.DB, organizationID uuid.UUID, userID uuid.UUID) (bool, error) {
```
Same as above. The organization ID for the current user is inside the claims.